duckduckgo/iOS
# Introduce 'Close tabs' option for Fire Button

Task/Issue URL: https://app.asana.com/0/856498667320406/1147552139577711
Tech Design URL:
CC:

**Description**:
Add option to Close Tabs without Clearing Data.

**Steps to test this PR**:
- Ensure 'Close Tabs' option work.
- Pixels (for new and old actions) should be fired so we can check engagement.
- Accessibility features around fire button should work.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
